content/blog/build-your-own-ilo-redfish-simulator.md

```diff
@@ -0,0 +1,146 @@
+
+When a Redfish client sends a GET request to the Redfish Mockup Server, it responds with the `index.json` file located in the folder of the requested endpoint as well as `200 OK` response code. If the target endpoint is not valid, the usual response codes `40X`  will be sent back.
+
+For POST, PUT and PATCH requests, the simulator performs limited verification of the query, modifies the requested endpoint and sends back a `204 No Content` status code with no associated body. For the same request, a real Redfish service performs additional verification and sends back a non-empty response body with a `20X` status code. 
+
+The different behavior of the simulator, compared to a real iLO 5 Redfish service can be illustrated with a PATCH request for modifying the `IndicatorLED` resource of a computer chassis. The authorized values for this parameter are defined by the DMTF in the Chassis Redfish schema. For an iLO 5 with firmware version 2.30, the [implemented schema](http://redfish.dmtf.org/schemas/v1/Chassis.v1_10_2.json#/definitions/IndicatorLED) specifies the following possible values: `unknown`, `Lit`, `Blinking`, `Off`. A physical iLO 5 complains if you supply a value different from what the schema proposes. However, the DMTF Redfish Mockup Server accepts any string as shown in the following screenshot. 
+
+The code of this picture sends a `PATCH` request with and invalid value (`Foo`) toward an iLO 5 simulator. The simulator performs the patch action and responds with status code `204`. The last command of shows that the action has been successfully performed.
+
+![PATCH an invalid property in the DMTF Redfish Mockup Server](/img/fooindicatorled.png "PATCH an invalid property in the DMTF Redfish Mockup Server")
+
+The same query against a physical iLO 5 returns a `400 Bad Request` status and a `@Message.ExtendedInfo` mentioning the faulty argument.
+
+![PATCH of an invalid property in a physical iLO 5](/img/fooindicatorledagainstilo5.png "PATCH of an invalid property in a physical iLO 5")
+
+### Querying the Mockup Server with iLOrest
+
+If you want to query your iLO simulator with [iLOrest](http://hpe.com/info/resttool) you have to extract the cache directory you saved during the mockup creation (see the "Preparing the use of iLOrest against your Mockup Server" paragraph above) and edit the `url` property of its two files: `index` and `<longUniqIdenfier>` to make them point to the simulator.
+
+On a Linux system, this operation can be done with the following commands:
+
+```Shell
+cd $iloCacheDir/cache/
+sed -i 's?\("url": "https://\)ilo-IP-physical"?\1ilo-IP-simulator"?' *
+```
+Once the iLOrest cache points to your mockup server, you can use this Redfish client tool to query the mockup. 
+
+> **NOTE** If you use the `login` iLOrest commands, the cache will be overwritten. If you use the `logout` command, the cache will be erased.
```

Comment:
In the previous note, you did not bold "NOTE" and you had a colon after it. You probably want to retain consistency, so pick one way or the other. I recommend, at least, placing a colon after "Note".